src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php

```diff
@@ -245,9 +247,10 @@ private function fixPhpUnitClass(Tokens $tokens, $startIndex, $endIndex)
             $tokens[$docBlockIndex] = new Token([T_DOC_COMMENT, $doc->getContent()]);
             $tokens->insertAt($braceIndex + 1, $newMethods);
 
-            $tokens[$braceIndex + $newMethods->getSize() + 1] = new Token([
+            $i1 = $braceIndex + $newMethods->getSize() + 1;
```

Comment:
Please rename the variable (e.g. `$whitespaceIndex`), `$i1` does not carry any useful information about the data is holds.

Reply:
Right, even used in a loop, all the other indexes are more verbosely named, it was only the $i below which might have irritated me a little.

Please consider your comment addressed and thanks for your thoughtful direction.